qiskit_nature/problems/second_quantization/__init__.py

```diff
@@ -39,7 +39,17 @@
 from .electronic import ElectronicStructureProblem
 from .lattice.lattice_model_problem import LatticeModelProblem
 from .vibrational.vibrational_structure_problem import VibrationalStructureProblem
-
+from ...deprecation import warn_deprecated, DeprecatedType, NatureDeprecationWarning
+
+warn_deprecated(
+    "0.5.0",
+    old_type=DeprecatedType.PACKAGE,
+    old_name="qiskit_nature.problems.second_quantization",
```

Comment:
The `lattice` stack partially moved to `hamiltonians`

Reply:
I changed the deprecation msg location but should we wait until the stack is moved to add the deprecation msgs. for this ?